robot/markers.py

```diff
@@ -76,7 +54,7 @@ def rot_y_deg(self):
         return math.degrees(self._rot_y_rad)
 
     @property
-    def distance_metres(self):
+    def distance_metres(self) -> float:
```

Comment:
If we were going full type system nuttery I'd advocate for having this return `Metres` as a newtype of floats. Ditto for degrees above.